### DISCUSSION TITLE
希望能更新使用文档

### DISCUSSION THREAD

**Discussion by jerrydiy** [ppoffice/hexo-theme-icarus#1229] [General]
addthis已经停止服务了2333
https://www.addthis.com/
这里能不能更新一下
https://ppoffice.github.io/hexo-theme-icarus/Plugins/Share/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E5%88%86%E4%BA%AB%E6%8C%89%E9%92%AE/

**Comment by ppoffice** [comment]
感谢提醒。文档已更新。
